applications/luci-app-ddns-go/po/zh_Hans/ddns-go.po

```diff
@@ -58,3 +58,7 @@ msgstr "跳过证书检查"
 #: applications/luci-app-ddns-go/htdocs/luci-static/resources/view/ddns-go.js:83
 msgid "Update interval"
 msgstr "更新间隔"
+
+#: applications/luci-app-ddns-go/htdocs/luci-static/resources/view/ddns-go.js:87
+msgid "DNS Server"
+msgstr "DNS地址"
```

Comment:
```suggestion
msgid "Custom DNS server"
msgstr "自定义 DNS 服务器"
```

use `./build/i18n-scan.pl` to update templates, then use `./build/i18n-update.pl` to sync translations.